rllib/models/experimental/tests/catalog.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+
+from gymnasium.spaces import Box
+
+from ray.rllib.models.experimental.catalog import Catalog
+
+
+class TestCatalog(unittest.TestCase):
+    def test_get_base_model_config(self):
+        """Tests if we can create a bunch of models from the base catalog class."""
+
+        # TODO (Artur): Add support for the commented out spaces
+        input_spaces = [
+            Box(-1.0, 1.0, (5,), dtype=np.float32),
+            # Box(-1.0, 1.0, (84, 84, 1), dtype=np.float32),
```

Comment:
nit: we should not interleave code with TODOs like this, just say 

```
# TODO (Artur): Add support for:
# item 1
# item 2
# ....
input_spaces = [Box(-1, 1, (5,), dtype)]
```

Reply:
Why?